storage/plugins/book-club/src/AffinityRepo.php

```
@@ -0,0 +1,582 @@
+            [$clubId, $userId]
+        );
+    }
+
+    /**
+     * Opt out AND immediately purge every stored pair involving the user,
+     * so the member disappears from everyone's list without waiting for
+     * the next recompute.
+     */
+    public function optOut(int $clubId, int $userId): bool
+    {
+        $ok = $this->exec(
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)